ksqldb-functional-tests/src/test/resources/query-validation-tests/protobuf.json

```diff
@@ -184,6 +184,105 @@
+        "CREATE STREAM OUTPUT WITH (value_format='PROTOBUF_NOSR', value_protobuf_nullable_representation='OPTIONAL') as SELECT * FROM input;"
+      ],
+      "inputs": [
+        {"topic": "input", "value": {"i": null, "l": 0, "d": null, "b": false, "s": null}},
+        {"topic": "input", "value": {"i": 0, "l": null, "d": 0.0, "b": null, "s": ""}}
+      ],
+      "outputs": [
+        {"topic": "OUTPUT", "value": {"i": null, "l": 0, "d": null, "b": false, "s": null}},
+        {"topic": "OUTPUT", "value": {"i": 0, "l": null, "d": 0.0, "b": null, "s": ""}}
+      ],
+      "post": {
+        "topics": {
+          "topics": [
+            {
+              "name": "input",
+              "keyFormat": {"format": "KAFKA"},
+              "partitions": 4,
+              "valueFormat": {"format": "PROTOBUF_NOSR", "properties": {"unwrapPrimitives": "true", "nullableRepresentation": "optional"}}
+            },
+            {
+              "name": "OUTPUT",
+              "keyFormat": {"format": "KAFKA"},
+              "partitions": 4,
+              "valueFormat": {"format": "PROTOBUF_NOSR", "properties": {"unwrapPrimitives": "true", "nullableRepresentation": "optional"}}
+            }
+          ]
+        }
+      }
+    },
+    {
+      "name": "protobuf_nosr primitives with null and nullable as wrapper",
+      "statements": [
+        "CREATE STREAM INPUT (K STRING KEY, i INTEGER, l BIGINT, d DOUBLE, b BOOLEAN, s STRING) WITH (kafka_topic='input', value_format='PROTOBUF_NOSR', value_protobuf_nullable_representation='WRAPPER');",
+        "CREATE STREAM OUTPUT WITH (value_format='PROTOBUF_NOSR', value_protobuf_nullable_representation='WRAPPER') as SELECT * FROM input;"
+      ],
+      "inputs": [
+        {"topic": "input", "value": {"i": null, "l": 0, "d": null, "b": false, "s": null}},
+        {"topic": "input", "value": {"i": 0, "l": null, "d": 0.0, "b": null, "s": ""}}
+      ],
+      "outputs": [
+        {"topic": "OUTPUT", "value": {"i": null, "l": 0, "d": null, "b": false, "s": null}},
+        {"topic": "OUTPUT", "value": {"i": 0, "l": null, "d": 0.0, "b": null, "s": ""}}
+      ],
+      "post": {
+        "topics": {
+          "topics": [
+            {
+              "name": "input",
+              "keyFormat": {"format": "KAFKA"},
+              "partitions": 4,
+              "valueFormat": {"format": "PROTOBUF_NOSR", "properties": {"unwrapPrimitives": "true", "nullableRepresentation": "wrapper"}}
+            },
+            {
+              "name": "OUTPUT",
+              "keyFormat": {"format": "KAFKA"},
+              "partitions": 4,
+              "valueFormat": {"format": "PROTOBUF_NOSR", "properties": {"unwrapPrimitives": "true", "nullableRepresentation": "wrapper"}}
+            }
+          ]
+        }
+      }
+    },
```

Comment:
Given that this PR is also adding `unwrapPrimitives` support for `PROTOBUF_NOSR`, can you also add the equivalent of this test? https://github.com/confluentinc/ksql/blob/master/ksqldb-functional-tests/src/test/resources/query-validation-tests/protobuf.json#L269

Reply:
Done